vector/src/main/java/im/vector/app/core/preference/PushRulePreference.kt

```diff
@@ -142,7 +143,7 @@ class PushRulePreference : VectorPreference {
                                     && safeRule.ruleId != RuleIds.RULE_ID_INVITE_ME
                                     && NOTIFICATION_NOISY_INDEX == index)
 
-                    if (NOTIFICATION_NOISY_INDEX == index) {
+                    if (NOTIFICATION_NOISY_INDEX == index && RuleIds.RULE_ID_ROOM_NOTIF != safeRule.ruleId) {
```

Comment:
Same remark, should be nice to have some comment to explain why this code is necessary.